src/__tests__/store/user/reducer.test.ts

```diff
@@ -0,0 +1,70 @@
+import {IUserState,UserActionTypes} from "../../../store/user/types";
+import {userReducer} from "../../../store/user/reducer";
+import { any } from "prop-types";
+
+
+const initialState: IUserState = {
+    username: null,
+    token:  null,
+    isRememberMe: false,
+    isLoggedIn: false
+};
+
+const UserState:IUserState = {
+    username: "string",
```

Comment:
instead of string, can you add sample data here? username: ```"joe"```

Reply:
Yes, we changed all the testing object with meaningful strings.